companies/tasks/report_processing.py

```
@@ -62,54 +198,119 @@ def extract_report_content_task(self, report_id: int) -> dict[str, Any] | None:
+    주의: RefineService는 뉴스 기사용이므로 보고서 정제 시 과도하게 내용이 제거될 수 있음.
+    정제 결과가 너무 짧으면 원본 내용(raw_content)을 사용하여 fallback 처리.
+    """
     if not data:
         return None
 
     report_id = data.get("report_id")
     try:
-        raw_content = data["raw_content"]
+        # Step 1에서 이미 섹션별 추출 완료
+        section_content = data.get("section_content", "")
+        tables_markdown = data.get("tables_markdown", "")
```

Ruff (0.14.13)

Local variable `tables_markdown` is assigned to but never used

Remove assignment to unused variable `tables_markdown`

(F841)